src/libsync/theme.h

```diff
@@ -565,6 +565,8 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
      */
     virtual bool showVirtualFilesOption() const;
 
+    virtual bool enforceVirtualFilesSyncFolder() const;
```

Comment:
Nit: Is the virtual really needed?

Reply:
I am going to have a look at why they are all virtual methods later
and for reference I suppose that is for historical reasons because that does not make sense in the current context